bin/codacy-analysis-cli.sh

```diff
@@ -31,6 +31,7 @@ run() {
     output_volume="--volume ${OUTPUT_DIRECTORY}:${OUTPUT_DIRECTORY}";
   fi
   local CODACY_ANALYSIS_CLI_VERSION="${CODACY_ANALYSIS_CLI_VERSION:-stable}"
+  docker pull codacy/codacy-analysis-cli:${CODACY_ANALYSIS_CLI_VERSION}
```

Comment:
although --pull flag is in docker run documentation, it was not available in the version I have installed locally. for this reason I included the pull as a separate command

![image](https://user-images.githubusercontent.com/36695843/130482438-60c3f80a-1558-41ad-b739-169edc4c6f37.png)

Reply:
Alright, consider some quoting here

```suggestion
  docker pull "codacy/codacy-analysis-cli:$CODACY_ANALYSIS_CLI_VERSION"
```